src/test/kotlin/com/ioki/lokalise/gradle/plugin/unit/UploadTranslationsTaskTest.kt

```diff
@@ -98,6 +157,7 @@ class UploadTranslationsTaskTest {
             .withArguments("uploadTranslations", "--configuration-cache", "--info")
             .buildAndFail()
 
+        println(result.output)
```

Comment:
```suggestion
```
No need for this 🙃

Reply:
Whoops! That should not have gotten in there.